theia-python-docker/Dockerfile

```diff
@@ -1,22 +1,43 @@
 ARG NODE_VERSION=10
-FROM node:${NODE_VERSION}-stretch
+FROM node:${NODE_VERSION}
 
+# Install Python 3 from source
 RUN apt-get update \
-    && apt-get install -y python python-dev python-pip \
-    && apt-get clean && rm -rf /var/cache/apt/* && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
+    && apt-get upgrade -y \
+    && apt-get install -y make build-essential libssl-dev \
+    && apt-get install -y libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+    && apt-get install -y libncurses5-dev  libncursesw5-dev xz-utils tk-dev \
+    && wget https://www.python.org/ftp/python/3.7.0/Python-3.7.0.tgz \
+    && tar xvf Python-3.7.0.tgz \
+    && cd Python-3.7.0 \
+    && ./configure \
+    && make -j8 \
+    && make install
 
-RUN pip install \
-    python-language-server \
-    flake8 \
-    autopep8
-
-ARG version=latest
+RUN apt-get update \
+    && apt-get install -y python-dev python-pip \
+    && apt-get install -y python3-dev python3-pip \
+    && apt-get install -y yarn \
+    && apt-get clean \
+    && rm -rf /var/cache/apt/* \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /tmp/*
 
+RUN adduser --disabled-password --gecos '' theia
```

Comment:
If we do not have sudo I think we need to get rid of the Theia user and have root as default in the container. else it will not be possible to install anything. 

You'll need to explicitly create the `/home/theia` directory however, like it was probably done in the previous version of the image.